lib/rules/no-invalid-double-slash-comments/index.mjs

```diff
@@ -47,18 +47,18 @@ const rule = (primary) => {
 					const i = index + offset;
 
 					ruleStringified ??= ruleNode.toString();
-					const lines = ruleStringified.slice(i).split('\n');
-					let comment = lines[0] ?? '';
 
-					if (lines.length > 1) comment += '\n';
+					const lines = ruleStringified.slice(i).split('\n');
```

Comment:
[suggest] I remember the existence of `context.newline`. Should we use it as much as possible like this?

```suggestion
					const lines = ruleStringified.slice(i).split(context.newline ?? '\n');
```

Reply:
In #7895, we confirmed that newline is always truthy.
A PR that updates the type is long overdue.

ref https://github.com/stylelint/stylelint/pull/5418#discussion_r1636001255